packages/rancher-cis-benchmark/charts/values.yaml

```diff
@@ -4,8 +4,8 @@
 
 image:
   cisoperator:
-    repository: rancher/cis-operator
-    tag: v1.0.3-rc2
+    repository: prachidamle/cis-operator
+    tag: a3
```

Comment:
This is a placeholder image tag to be replaced after cis-operator changes are merged